tests/tensor_search/test_api.py

```diff
@@ -51,6 +51,60 @@ def test_add_or_replace_documents_tensor_fields(self):
             self.assertEqual(response.status_code, 200)
             mock_add_documents.assert_called_once()
 
+    def test_add_or_replace_documents_normalize_embeddings_true_zero_vector_passed(self):
```

Comment:
These could be moved to the actual marqo-api-tests repo

Reply:
Raised a PR for changes in marqo-api-tests repo: https://github.com/marqo-ai/marqo-api-tests/pull/97